### PR TITLE
add missing step: Enable app verification

### PR DESCRIPTION
**Pull request by MacKentoch**
Without this step, an error will be thrown.

This is just a little configuration as clearly documented in firebase documentation.

### CODE REVIEW

**Comment by MacKentoch** [comment]
By the way: thank you for this library 👍

File: docs/auth/ios.md [hunk 1]
```diff
@@ -12,4 +12,12 @@ pod 'Firebase/Auth', '~> {{ ios.firebase.auth }}'
...
 
-?> If you plan to make use of email sign-in links then you'll also need to follow the [Dynamic Links for iOS installation guide](version /links/ios).
\ No newline at end of file
+## Enable app verification
+
+Phone authentication leaves you 2 choices:
+- Silent APNs notifications
+- reCAPTCHA verification
+
+You need to follow Firebase related documentation to complete this step [Firebase-phone-auth](https://firebase.google.com/docs/auth/ios/phone-auth) otherwise app verification will throxw an error.
```
Comment (Salakar) [inline docs/auth/ios.md]: Typo `throxw`

File: docs/auth/ios.md [hunk 2]
```diff
@@ -12,4 +12,12 @@ pod 'Firebase/Auth', '~> {{ ios.firebase.auth }}'
 
 Run `pod update`.
 
-?> If you plan to make use of email sign-in links then you'll also need to follow the [Dynamic Links for iOS installation guide](version /links/ios).
\ No newline at end of file
+## Enable app verification
```
Comment (Salakar) [inline docs/auth/ios.md]: Maybe a `(optional for Phone Authentication)` at the end of the header?

**Comment by Salakar** [comment]
Thanks for this! :)

**Pull request merged**